[MTM-53671] Alarms documentation update (#1884)
SoftwareAG/c8y-docs: content/device-management-application/monitoring-and-controlling-devices-bundle/working-with-alarms.md

{{< product-c8y-iot >}} displays alarms at the level of individual devices and across all devices:

* To check the alarms for all devices, click **Alarms** in the **Overview** menu in the navigator.
* To check the alarms of a particular device, switch to the **Alarms** tab in the details of this device.

![Alarms page](/images/users-guide/DeviceManagement/devmgmt-alarms-view.png)

By default,

* only unresolved alarms are shown. If you turn on **Show cleared alarms** in the **Filter alarms** dropdown menu at the top, you will see the entire alarm history.

> * The alarms list is automatically refreshed every 30 seconds. Click **Disable auto-refresh** to stop the alarms list from refreshing automatically. ![Alarms refresh interval](/images/users-guide/DeviceManagement/devmgmt-alarms-refresh-interval.png)

Reviewer: I don´t believe that this image is needed. Remember that we aim at reducing the overall number of images to those which are of a real value to the user to avoid unnecessary maintenance effort if the design changes.

Author: removed

|MINOR|The device has a problem that may be fixed.
|WARNING|There is a warning.

> In the top menu bar, a **Filter Alarms** button is provided to filter by severity. By clicking a filter button, you can check or uncheck severities that you want to show or hide, and after that, click **Apply**.

Reviewer: ```suggestion
In the top menu bar, a **Filter alarms** dropdown list is provided. Select the severities to be shown and then click **Apply**.
```

Author: done

{{< c8y-admon-info >}}

> The number provided in the filter menu refers to the number of active alarms for the given severity, as opposed to the counter provided as blue circle next to an active alarm, which shows the number of times this same alarm has occurred (see also the table below).

Reviewer: ```suggestion
The number provided in the filter dropdown refers to the number of active alarms for the given severity, as opposed to the counter provided as blue circle next to an active alarm, which shows the number of times this same alarm has occurred (see also the table below).
```

Author: done

</tr>
<tr>
<td align="left">Device</td>
<td align="left">The name of the device.</td>
</tr>
</tbody>
</table>

> Select an alarm by clicking on the alarm in the **Alarms list** that you are interested in, to display further details about the alarm.

Reviewer: ```suggestion
To display further details about an alarm, click on the respective alarm in the **Alarms list**.
```

> * **Status**: Provides further information on the alarm status and showing the type of the alarm. The type info is used for duplicating alarms and for configuring the priority of alarms in [Alarm mapping](/standard-tenant/alarm-mapping/).

Reviewer: ```suggestion
* **Status**: Provides further information on the alarm status and shows the type of the alarm. The type info is used for duplicating alarms and for configuring the priority of alarms in [Alarm mapping](/standard-tenant/alarm-mapping/).
```
